src/com/epam/javatrainings/pizzeria/Customer.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.javatrainings.pizzeria;
+
+public class Customer {
+    public int id;
```

Comment:
please make fields private and create for them getters/setters if needed

Reply:
Ok. Thank you Sevak jan fo review